.worlddriven.ini-example

```diff
@@ -1,3 +1,4 @@
 [DEFAULT]
-baseMergeTimeInHours = 120
-perCommitTimeInHours = 120
+baseMergeTimeInHours = 240
+perCommitTimeInHours = 0
+merge_method = squash
```

Comment:
I might be wrong here, but I think these are in the opposite files to what they should be. Shouldn't the config for this repository (these values here) be in .worlddrive.ini and the other ones in here as the example?

Reply:
You are absolutely right, thx, nice catch, fixed